CDCgov/prime-reportstream
# Experience/6169 file handler reducer

Moves state based functionality for file handlers (validation and upload pages) into a reducer hook. 

TODO:

- [x] move constants?
- [x] move csv local error checker?

## Test Steps:

No user facing changes here. Review test steps from https://github.com/CDCgov/prime-reportstream/pull/6079 to make sure the uploader / validator still works.

## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #issue 6169

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue
